rskj-core/src/main/java/org/ethereum/core/Block.java

```diff
@@ -660,4 +711,10 @@ public void flushRLP() {
         this.rlpEncoded = null;
         this.parsed = true;
     }
+
+    public static class SealedBlockException extends RuntimeException {
```

Comment:
Creating this exception on it's own file would be better. That's a way to avoid unnecessary growth on this already too long class file.